README.md

```diff
@@ -44,6 +44,15 @@ unless real values don't help users know what to change.
 -->
 
 ```hcl
+provider "restapi" {
+  uri = "https://private.resource-controller.cloud.ibm.com"
```

Comment:
perhaps show the public endpoint in the module usage but leave a code comment which shows ehat private endpoint is